declarations.d.ts

```diff
@@ -63,6 +63,29 @@ interface IiTunesConnectApplication {
 }
 
 /**
+ * Describes configuration settings that modify the behavior of some methods.
+ */
+interface IConfigurationSettings {
+	/**
+	 * This string will be used when constructing the UserAgent http header.
+	 * @type {string}
+	 */
+	userAgentName: string;
```

Comment:
I value the semantics over the usage - semantically you're setting part of the User-Agent http header and not the clientName.
Besides we've already got a `CLIENT_NAME` property in the `$staticConfig` and it would be weird to reuse the same lexical word for a different semantic meaning